pkg/crypto/webkms/remotecrypto.go

```diff
@@ -586,6 +618,195 @@ func (r *RemoteCrypto) UnwrapKey(recWK *crypto.RecipientWrappedKey, keyURL inter
 	return keyBytes, err
 }
 
+// SignMulti will create a BBS+ signature of messages using the signer's private key handle found at signerKeyURL.
+// returns:
+// 		signature in []byte
+//		error in case of errors
+func (r *RemoteCrypto) SignMulti(messages [][]byte, signerKeyURL interface{}) ([]byte, error) {
+	startSign := time.Now()
+	destination := fmt.Sprintf("%s", signerKeyURL) + bbsSignURI
+
+	var encMessages []string
+	for _, msg := range messages {
+		encMessages = append(encMessages, base64.URLEncoding.EncodeToString(msg))
+	}
+
+	sReq := bbsSignReq{
+		Messages: encMessages,
+	}
+
+	httpReqBytes, err := r.marshalFunc(sReq)
+	if err != nil {
+		return nil, fmt.Errorf("marshal signature request for BBS+ Sign failed [%s, %w]", destination, err)
+	}
+
+	resp, err := r.postHTTPRequest(destination, httpReqBytes)
+	if err != nil {
+		return nil, fmt.Errorf("posting BBS+ Sign message failed [%s, %w]", destination, err)
+	}
+
+	// handle response
+	defer closeResponseBody(resp.Body, logger, "BBS+ Sign")
+
+	respBody, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, fmt.Errorf("read signature response for BBS+ Sign failed [%s, %w]", destination, err)
+	}
+
+	httpResp := &signResp{}
+
+	err = r.unmarshalFunc(respBody, httpResp)
+	if err != nil {
+		return nil, fmt.Errorf("unmarshal signature for BBS+ Sign failed [%s, %w]", destination, err)
+	}
+
+	keyBytes, err := base64.URLEncoding.DecodeString(httpResp.Signature)
+	if err != nil {
+		return nil, err
+	}
+
+	logger.Infof("overall BBS+ Sign duration: %s", time.Since(startSign))
```

Comment:
Todo: these shouldn't be info logs.

Reply:
we'll switch to Debugf once perf testing is complete in a future commit